### PR TITLE
Rewrite README to remove obsolete stuff

### PR DESCRIPTION
**Pull request by theletterf**
Remove references to old stuff and add links to docs and license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,107 +1,32 @@
...
+Splunk Observability Cloud and the Splunk Distribution of OpenTelemetry Collector
+can ingest spans in the following formats:
 
-For customers who have not yet instrumented their code, our default
-recommendation is to use OpenTracing to instrument since it has a growing
-ecosystem of library owners & frameworks instrumenting their code with it & use
-Jaeger tracer libraries to export spans to us - both of these are CNCF projects
-with a rapidly growing community behind them.
+- OpenTelemetry Protocol or OTLP (Preferred)
+- Jaeger gRPC and Thrift
```
Comment (Kielek) [inline README.md:16]: Jeager protocol was deprecated, I am not sure if we should promote it.

Comment (theletterf) [reply README.md:16]: Good point, though at the same it's true that we can still ingest it...

Comment (theletterf) [reply README.md:16]: Let me remove it for now then :)

**Comment by theletterf** [comment]
PTAL

File: README.md [hunk 2]
```diff
@@ -1,107 +1,31 @@
...
-       alt="Jaeger Tracing" /></a>
-  &nbsp;&nbsp;
-  <a href="https://zipkin.io">
-  <img src="https://avatars3.githubusercontent.com/u/11860887?s=100&v=4"
-       alt="ZipKin" /></a>
-    &nbsp;&nbsp;
-  <a href="https://opencensus.io">
-  <img src="https://avatars3.githubusercontent.com/u/26944525?s=100&v=4"
-       alt="OpenCensus" /></a>
-</p>
```
Comment (breedx-splk) [inline README.md:34]: 🤘🏻 😁 🤘🏻

File: README.md [hunk 3]
```diff
@@ -1,107 +1,31 @@
...
+Splunk Observability Cloud and the Splunk Distribution of OpenTelemetry Collector
+can ingest spans in the following formats:
```
Comment (breedx-splk) [inline README.md:13]:
```suggestion
Splunk Observability Cloud and the Splunk Distribution of OpenTelemetry Collector
can ingest tracing data in the following formats:
```

The list is not comprehensive but the statement reads like it is. Is there a way to say "these are the relevant/interesting formats we cover in this repo" or something?

**Review by breedx-splk — APPROVED** [review]
Thanks so much!

File: README.md (hunk 3 above)
Comment (theletterf) [reply README.md:13]: Sure! Adding

**Review by pellared — APPROVED** [review]
We talk only about "tracing" while we do more (e.g. metrics, profiling). Maybe we should change the wording a little? But in a separate PR 😉

**Pull request merged**